### DISCUSSION TITLE
Can add more supported platforms for the cocoapod spec?

### DISCUSSION THREAD

**Discussion by wu-hui** [nlohmann/json#2644] [Q&A] [answered]
I am trying to use the library for my project using cocoapods, which runs on iOS/macOS/tvOS. I can get iOS working, but other platforms are not "supported" in the pod spec.

As I understand, there is no reason why it won't work there. Can we update the pod spec? Or do you prefer us to use local spec in this case?

**Comment by nlohmann** [comment, marked as answer]
I am not maintaining that spec myself, but would be happy for PRs.

**Reply by wu-hui** [reply]
I can certainly make a PR, but do you know where I should make the PR to?

**Reply by nlohmann** [reply]
As the README states, please file issues here: https://bitbucket.org/benman/nlohmann_json-cocoapod/issues
